### ISSUE TITLE
Brizy edit not loading, keep spinning in the page 

### ISSUE THREAD

**Issue by AlexandruRaicu** [ThemeFuse/Brizy#1355]
The issue is happening with a client of us

He has the last version of the plugin which at the moment is 1.0.76
WordPress 5.2

![a](https://user-images.githubusercontent.com/15956680/58073105-6e7f3d00-7baa-11e9-8193-67ad8a986027.png)

**Comment by MaxZiebell** [comment]
I can't edit blocks anymore. Brizy… please get back to us!

**Comment by NataliaBB** [comment]
Hello @AlexandruRaicu 
Are you able to install the latest  1.0.77 version to your client? 

Natalia

**Comment by dinuionut** [comment]
Hi @NataliaBB, @AlexandruRaicu is part of my team, I just rechecked for him because it looks like he missed your reply.
Current version the client is using is the latest, 1.0.81. No errors are showing up in console.

**Comment by dinuionut** [comment]
Issue can be closed, we solved it on our side.

Some other function of ours was trying to redirect the user to a different page in the frontend, while Brizy was trying to redirect the user to the page he was going to edit, which resulted in a redirect loop in background.

I don't know if there's anything you should also do to prevent such situations with other plugins/themes, but for now we'll add a condition to prevent such redirects while in the Brizy edit screen.

Thanks!

**Comment by EmergenceCreativeSpace** [comment]
Every single one of the pages I created using brizy are now messed up.
Not sure what's up. But it isn't good!
Here is a link to a page that is a great example of what is happening.

https://emergencecreativespace.com/ecs-community-sign-up/

Also, I can't find the brizy editor on my pages or posts anymore. So I am guessing that is part of the problem. When I tried to get the Brizy plugin it tells me that the file is already there.
